style.css

```diff
@@ -422,3 +426,151 @@ form {
   /* Internet Explorer 10-11 */
   color: black;
 }
+
+/* Hide desktop version element */
+.only-desktop {
+  display: none;
+}
+
+/* Use media queries to make my desktop version look like the templateIn figma
+(Desktop: 768px or wider). */
+@media only screen and (min-width: 768px) {
+  /* Standard CSS rules */
+  html {
+    box-sizing: border-box;
+  }
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+
+  /* Hide the header menu button and welcome logo */
+  header > a,
+  header > img {
+    display: none;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  /* Add the headline background  */
+  #headline {
+    background-image: url("img/desktop-bg/Header-llustration.png");
+    background-size: 100% 100%;
+    padding: 1em 10em;
+  }
+
+  /* style the navbar */
+  nav {
+    margin-top: -1em;
+  }
+
+  nav a {
+    text-decoration: none;
+    color: #172b4d;
+    font-size: large;
+    font-weight: 550;
+  }
+
+  nav a:focus {
+    color: #008552;
+  }
+
+  .only-desktop {
+    display: block;
+  }
+
+  .primary > h1 {
+    font-size: 275%;
+  }
+
+  .supporting-text > p {
+    font-size: 150%;
+  }
+
+  .disabled-btn {
+    padding-top: 2em;
+    padding-bottom: 7em;
+  }
+
+  /* Design the grid container for works section */
+  .grid-layout {
+    width: 95%;
+    margin: 2em;
+    -ms-grid-columns: repeat(auto-fit, minmax(300px, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    gap: 5px;
+  }
```

Comment:
- [ ] [Optional] As suggested by the previous reviewer, Kindly make sure to display your cards in 2 rows of 3 columns in the desktop version. Right now as the desktop screen size increases, the number of columns in your rows varies. It would be great to rectify this issue.

Reply:
Done